spec/tbd_tests_spec.rb

```diff
@@ -1424,6 +1429,159 @@
+        next unless id.include?("Sub Surface")
+
+        type = edge[:type].to_s.downcase
+        subs[id] = [] unless subs.key?(id)
+        subs[id] << type
+      end
+    end
+
+    # "Sub Surface 7" ~ "Sub Surface 8" ~ "Sub Surface 6"
+    # "Sub Surface 3" ~ "Sub Surface 5" ~ "Sub Surface 4"
+    # "Sub Surface 1" ~ "Sub Surface 2"
+    subs.each do |id, types|
+      expect(types.size).to eq(2)
+
+      kid = model.getSubSurfaceByName(id)
+      expect(kid.empty?).to be(false)
+      kid = kid.get
+      dad = kid.surface
+      expect(dad.empty?).to be(false)
+      dad = dad.get
+      nom = dad.nameString
+      expect(surfaces.key?(nom)).to be(true)
+      loss = surfaces[nom][:heatloss]
+      less = 0.200    # jamb PSI factor (in W/K per meter)
+      # Sub Surface 6 : 0.496           (height in meters)
+      # Sub Surface 8 : 0.488
+      # Sub Surface 7 : 0.497
+      # Sub Surface 5 : 1.153
+      # Sub Surface 3 : 1.162
+      # Sub Surface 4 : 1.163
+      # Sub Surface 1 : 0.618
+      # Sub Surface 2 : 0.618
+
+      case id
+      when "Sub Surface 5"
+        expect(types.include?("jamb")).to be(false)
+        expect(types.include?("transition")).to be(true)
+        less *= (2 * 1.153) # 2x transitions; no jambs
+      when "Sub Surface 8"
+        expect(types.include?("jamb")).to be(false)
+        expect(types.include?("transition")).to be(true)
+        less *= (2 * 0.488) # 2x transitions; no jambs
+      when "Sub Surface 6"
+        expect(types.include?("jamb")).to be(true)
+        expect(types.include?("transition")).to be(true)
+        less *= (1 * 0.496) # 1x transition; 1x jamb
+      when "Sub Surface 7"
+        expect(types.include?("jamb")).to be(true)
+        expect(types.include?("transition")).to be(true)
+        less *= (1 * 0.497) # 1x transition; 1x jamb
+      when "Sub Surface 3"
+        expect(types.include?("jamb")).to be(true)
+        expect(types.include?("transition")).to be(true)
+        less *= (1 * 1.162) # 1x transition; 1x jamb
+      when "Sub Surface 4"
+        expect(types.include?("jamb")).to be(true)
+        expect(types.include?("transition")).to be(true)
+        less *= (1 * 1.163) # 1x transition; 1x jamb
+      when "Sub Surface 1"
+        expect(types.include?("jamb")).to be(true)
+        expect(types.include?("transition")).to be(true)
+        less *= (1 * 0.618) # 1x transition; 1x jamb
+      when "Sub Surface 2"
+        expect(types.include?("jamb")).to be(true)
+        expect(types.include?("transition")).to be(true)
+        less *= (1 * 0.618) # 1x transition; 1x jamb
+      end
+
+      # 'dads[ (parent surface identifier) ]' holds TBD-estimated heat loss
+      # from major thermal bridging (in W/K per m) in the initial case. The
+      # substitution of 1x or 2x subsurface jamb edge types to (mild)
+      # transition(s) reduces the (revised) heat loss in the second case.
+      expect(loss + less).to be_within(TOL).of(dads[nom])
```

Comment:
Here, "less" corresponds to the difference in heat loss, in W/K (not W/K per m - _my bad_), from major thermal bridging before/after having TBD reset window **jambs** as (mild) **transitions** as a result of a user-set proximity tolerance of e.g. 100mm.